ifrs17/Import/Importers.ipynb

```diff
@@ -299,9 +299,9 @@
         {
             "cell_type": "markdown",
             "source": [
-                "## Parse Main Tab and return Args",
+                "## Parse Main Tabs",
                 "\n",
-                "\nThe main table of our custom import formats contains the information which are required to identify the data partition. These information are temporarily stored in [Args](../DataModel/DataStructure#args) and used in the next methods."
+                "\nThe main table of our custom import formats contains the information (Year, Month, Scenario) which are required to identify unpartition data. These information are temporarily stored in [Args](../DataModel/DataStructure#args) and used in the next methods."
```

Comment:
Improve this description:
- Text is repeated in the next markdown 
- improve description: now it seems that the main only contains  (Year, Month, Scenario). Start general (standard main section from transactional data) and add details one at the time (yield curve). You don't have to describe every little detail such as why we extract common part into methods. 
- I would reset the change about unpartition data.